letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
I think it is fair to cover both scenarios.

In your scenarior it is not expected to have any collision and in our scenario (multi-database) it is not expected too. Each challenge is unique. The only collateral effect could be to renew a certificate twice, but this is our sysadmin problem. Be have to care about to have letsencrypt installed only in one DB